src/app/view/components/_common/dot-action-button/dot-action-button.module.ts

```diff
@@ -0,0 +1,23 @@
+import { DotActionButtonComponent } from './dot-action-button.component';
```

Comment:
If you are going to use an old component that doesn't have it own module, please create it.